docs/source/notes/randomness.rst

```diff
@@ -121,3 +123,38 @@ CUDA RNN and LSTM
 In some versions of CUDA, RNNs and LSTM networks may have non-deterministic behavior.
 See :meth:`torch.nn.RNN` and :meth:`torch.nn.LSTM` for details and workarounds.
 
+CUDA random seed
+----------------
+
+Also it is recommended to set CUDAs random seed::
+
+    torch.cuda.manual_seed_all(0)
+    torch.cuda.manual_seed(0)
+
+Python
+......
+
+For custom operators, you might need to set python seed as well::
+
+    inport random
+    random.seed(0)
+
+DataLoader
+..........
+
+DataLoader will reseed workers following :ref:`data-loading-randomness` algorithm.
+Use :meth:`worker_init_fn` to preserve reproducibility::
+
+    def seed_worker(worker_id):
+        worker_seed = torch.initial_seed() % 2**32
+        numpy.random.seed(worker_seed)
```

Comment:
Maybe add a comment here on why there is no `torch.manual_seed()` ?

Reply:
I think in some cases users might want to use torch.manual_seed(), lets just leave it as is so they have idea where to look, and they can elaborate solution later on.